ONSdigital/ssdc-rm-response-operations
# 3230 - Add info when adding existing user to group

# Motivation and Context

When adding a user's email to a group, if the email already exists, it has been recommended that we display a more informative message to say that the email has already been added. 

There are security implications of exposing this information, but this is an internal system, so it's been decided to go ahead. Perhaps a revisit to this in future would be worthwhile, though.

# What has changed



Updated to check against existing emails

# How to test?



Try adding an existing user to a group. The error message should hopefully make sense.

# Links



https://trello.com/c/gwmy3fEM/3230-rops-add-user-validation-messages-need-improvement-when-adding-an-already-existing-user-5
